src/CKAN.jl

```diff
@@ -0,0 +1,60 @@
+struct CKAN <: DataRepo
+end
+
+base_url(::CKAN) = "https://demo.ckan.org/api/3/action/package_show?id="
+
+function description(repo::CKAN, mainpage)
+    desc = mainpage["notes"]
+    authors = [mainpage["author"], mainpage["maintainer"] * " (Maintainer)"]
+    author = format_authors(authors)
+    license = mainpage["license_title"]
+    rawdate = Dates.Date(mainpage["metadata_created"][1:10], "yyyy-mm-dd")
+    date = Dates.format(rawdate, "U d, yyyy")
+    dataset = mainpage["name"]
+    
+    final = escape_multiline_string("""
+    Author: $(author)
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+
+    Please cite this dataset:
```

Comment:
i would remove the `Please cite` section from CKAN, it really belong to data that is linked to accademic research.
CKAN is for goverment publications, and the conventions are different.
There is no paper to cite and the datasets don't e.g. have a DOI number.
The grey literature people probably have something to say about it, but for now it is beyond the scope of the project I think.
They top matter with the info is enough